pkg/sdkstats/sdkstats.go

```diff
@@ -130,33 +139,52 @@ func GetRepoDetails(client *github.Client, owner string, name string) map[string
 	repoMap["CreatedAt"] = repos.GetCreatedAt().String()
 	repoMap["PushedAt"] = repos.GetPushedAt().String()
 	repoMap["TotalCommits"] = strconv.Itoa(total)
-	return repoMap
+	repoMap["Language"] = repos.GetLanguage()
+	return repoMap, nil
 
 }
 
-func getversion(codeResults github.CodeResult, projectType string) int {
+//getVersion takes each Code result from a search output, and searches for Fragments comtaining VERSION, and returns the same.
+//The Fragments Text Macthes used are specific to Operator-SDK Version Pattern.
+func getversion(codeResults github.CodeResult, projectType string) string {
```

Comment:
Use camel case for identifiers.
```suggestion
func getVersion(codeResults github.CodeResult, projectType string) string {
```